### ISSUE TITLE
Pypi wheel package for apache-airflow-upgrade-check 1.1.0 is wrong

### ISSUE THREAD

**Issue by oost** [apache/airflow#13367]
**Apache Airflow version**: apache-airflow-upgrade-check 1.1.0

**Environment**:

- **Cloud provider or hardware configuration**: N/A
- **OS** (e.g. from /etc/os-release): N/A
- **Kernel** (e.g. `uname -a`): N/A
- **Install tools**: pip
- **Others**: N/A

**What happened**:

Content of the two files on [pypi](https://pypi.org/project/apache-airflow-upgrade-check/#files) is different. The tar file contains the code from tag upgrade-check/1.1.0 but it seems the wheel file contains code from tag upgrade-check/1.0.0. For example, the rule invalid_kubernetes_configs.py is present in the wheel file while absent in the [repo at the 1.1.0 tag](https://github.com/apache/airflow/tree/upgrade-check/1.1.0/airflow/upgrade/rules) as well as in the tar file on pypi. 

Am I missing something?

**Comment by potiuk** [comment]
I confirm that the two files are different but I think it was some accidental problem while preparing the .whl package.
I compared the contents and checkd also the sources in the official ASF SVN: https://archive.apache.org/dist/airflow/upgrade-check/1.1.0/ and it seems that the extra file you mentioned `invalid_kubernetes_configs.py` is the ONLY difference between the packages. All the rest is as-expected and matches the 1.1.0 tag.

My guess is that the whl file was first wrongly prepared - it was built first using the 1.0.0 label and then re-created with 1.1.0 which would explain why the file is there.

Does it cause any problems for you? Does it fail the rule or causes some other problems? 


Just in case - You know that you can also disable some rules by configuration as a workaround: 

https://pypi.org/project/apache-airflow-upgrade-check/

```
airflow upgrade_check --config=/files/upgrade.yaml
```

the configuration file should be a proper yaml file similar to this one:

```
ignored_rules:
  - InvalidKubernetesConfigRule
```

**Comment by potiuk** [comment]
cc: @kaxil @ashb -> I guess something to pay attention to with 1.2.0 release. I am not sure if my guess was ok, or whether there was some other reason for the accidental inclusion.

**Comment by oost** [comment]
Thanks. That works. 

Yes, I have been using a config file to ignore rules but couldn't find which rule to ignore as it was not there in the 1.1.0 tag. This will do.

**Comment by potiuk** [comment]
Reopened to make sure we check it when we release next time

**Comment by kaxil** [comment]
Fixed in apache-airflow-upgrade-check 1.2.0
